src/app/filter/filter.component.ts

```diff
@@ -88,6 +88,13 @@ export class FilterComponent {
         }
         this.inputToFocus = undefined;
       });
+    } else {
+      if (this.state && this.state.visible) {
+        const ft = this.filterTerm;
+        if (ft) {
+          ft.nativeElement.focus();
+        }
```

Comment:
Couldn't you replace these four lines by something like 
`this.filterTerm?.nativeElement.focus();`?
Or is optional chaining frowned upon?

Reply:
OK. Let's leave it like it is then.